sys-kernel/coreos-firmware/coreos-firmware-99999999.ebuild

```diff
@@ -115,6 +115,10 @@ src_prepare() {
 }
 
 src_install() {
-	insinto /lib/firmware/
-	doins -r *
+	local list=$(find -type f) || die
+
+	if [[ -n ${list} ]] ; then
+		insinto /lib/firmware/
+		doins -r ${list}
+	fi
```

Comment:
@marineam The current arm64 kernel config is pretty minimal, so needs no firmware.  In the general case there will be firmware files for arm64.  I think is too much to have conditional logic, etc.  I'll try to enable something in the arm64 kernel config that needs firmware.